apps/src/weblab/WebLab.js

```diff
@@ -135,6 +142,18 @@ WebLab.prototype.init = function (config) {
     isProjectLevel: !!config.level.isProjectLevel,
   });
 
+  function onAddFileHTML() {
+    this.brambleHost.addFileHTML();
+  }
+
+  function onAddFileCSS() {
+    this.brambleHost.addFileCSS();
+  }
+
+  function onAddFileImage() {
+    dashboard.assets.showAssetManager(null, 'image', this.loadCurrentAssets.bind(this), !this.studioApp_.reduxStore.getState().pageConstants.is13Plus);
```

Comment:
Nits, unrelated to your current change:

* Instead using the global `dashboard.assets.showAssetManager` we should probably be importing that method:

  ```javascript
  import showAssetManager from '@cdo/apps/code-studio/assets/show';
  ```

  However, it looks like none of the six existing usages do this, and it might be better to make this change all together in its own PR.  I think there are also test-stub implications.

* To follow convention, we should reorder the arguments for [`showAssetManager`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/assets/show.js#L16) to put the callback last, and should document the `showUnderageWarning` parameter.